net-libs/libwebsockets/libwebsockets-4.0.1-r1.ebuild

```diff
@@ -12,14 +12,16 @@ SRC_URI="https://github.com/warmcat/libwebsockets/archive/v${PV}.tar.gz -> ${P}.
 LICENSE="MIT"
 SLOT="0/16" # libwebsockets.so.16
 KEYWORDS="~amd64 ~arm ~x86"
-IUSE="access-log caps cgi client dbus generic-sessions http-proxy http2 ipv6
-	+lejp libev libevent libressl libuv peer-limits server-status smtp socks5
-	sqlite3 ssl static-libs threads zip"
+IUSE="access-log caps cgi client dbus +dir +file-ops generic-sessions http-proxy +http-uncommon-headers
+	http2 ipv6 +lejp libev libevent libressl libuv mbedtls peer-limits server-status smtp socks5 sqlite3
+	ssl static-libs testapps threads zip"
 
 REQUIRED_USE="
 	generic-sessions? ( smtp sqlite3 )
 	http-proxy? ( client )
 	smtp? ( libuv )
+	ssl? ( ?? ( libressl mbedtls ) )
+	testapps? ( static-libs )
```

Comment:
No.  testapps must link libraries dynamically.  This is policy.

Sorry, can't let this one slide.  If you need to, engage with Andy.